### PR TITLE
Added editableprofile

### PR DESCRIPTION
**Pull request by sirStroam**
Requires new column on user called bio

### CODE REVIEW

File: lib/core.php [hunk 1]
```diff
@@ -408,3 +415,34 @@ function updateGameVersionsCached($modid) {
...
+        $entype = "rc4";
+        $magickey = "VintageStory";
+        $usertoken = openssl_encrypt($stringtoencode, $entype, $magickey);
+		return rtrim($usertoken, "==");
+	}
+
+	function getUserFromToken($usertoken, $dbcon) {
+		$returnuser = null;
+		$entype = "rc4";
+        $magickey = "VintageStory";
```
Comment (tyronx) [inline lib/core.php:431]: some indentation issues here

File: lib/core.php [hunk 2]
```diff
@@ -408,3 +415,34 @@ function updateGameVersionsCached($modid) {
 		
 		if (count($tags)>0) $con->Execute("insert into modversioncached values " . implode(",", $inserts));
 	}
+
+	function getUserToken($userid, $joindate) {
```
Comment (tyronx) [inline lib/core.php:419]: better: genUserToken or createUserToken

File: lib/core.php [hunk 3]
```diff
@@ -408,3 +415,34 @@ function updateGameVersionsCached($modid) {
...
+		$paduserid = str_pad($userid, 8, '0', STR_PAD_LEFT);
+        $stringtoencode = "$joindate/$paduserid";
+        $entype = "rc4";
+        $magickey = "VintageStory";
+        $usertoken = openssl_encrypt($stringtoencode, $entype, $magickey);
+		return rtrim($usertoken, "==");
+	}
+
+	function getUserFromToken($usertoken, $dbcon) {
```
Comment (tyronx) [inline lib/core.php:428]: better: getUserByToken

File: lib/core.php (hunk 1 above)
Comment (sirStroam) [reply lib/core.php]: Formatted core.php

File: lib/core.php (hunk 3 above)
Comment (sirStroam) [reply lib/core.php]: Changed to getUserByToken

File: lib/config.php [hunk 4]
```diff
@@ -7,9 +7,9 @@
 // adding "127.0.0.1	stage.mods.vintagestory.at"  to your hosts file
 if (strstr($_SERVER["SERVER_NAME"], "stage.mods.vintagestory.at")) {
 	$config["database"] = "moddb";
-	$config["databasehost"] = "localhost";
-	$config["databaseuser"] = "root";
-	$config["databasepassword"] = "";
+	$config["databasehost"] = "127.0.0.1";
+	$config["databaseuser"] = "vsmoddb";
+	$config["databasepassword"] = "vsmoddb";
```
Comment (tyronx) [inline lib/config.php:12]: please leave these values untouched

Comment (sirStroam) [reply lib/config.php]: Reversed changes to this file.

**Comment by tyronx** [comment]
thanks for the PR!

**Pull request merged**